CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Example to demonstrate how to create vertical BarSeries
+- Add a public method `AddTypeface` to `SkiaRenderContext` to allow manually adding typefaces to the cache
```

Comment:
I think this line is obsolete now, otherwise this looks good to me now!

Reply:
true, I can remove it later this week.